crds/memorystore_v1alpha1_memorystoreinstance.yaml

```diff
@@ -318,56 +495,59 @@ spec:
                 description: ObservedState is the state of the resource as most recently
                   observed in GCP.
                 properties:
-                  authorizationMode:
```

Comment:
@yuwenma @justinsb Wonder if we should release note a fundamental change to this alpha resource.